src/main/java/com/hivemq/client/internal/checkpoint/Confirmable.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 dc-square and the HiveMQ MQTT Client Project
```

Comment:
Do you need to update your cp notice?

Reply:
Yes, will be updated to
`Copyright 2018-present HiveMQ and the HiveMQ Community`
but this will happen if all current PRs are merged, so it only needs to be done once.